File: server/router/common/common_db.go
package commom

import (
	v1 "go-cms/api/v1"
	"go-cms/middleware"

	"github.com/gin-gonic/gin"
)

type CommonDbRouter struct {
}

// InitCommDbRouter 初始化 QuickEdit 路由信息
func (s *CommonDbRouter) InitCommonDbRouter(Router *gin.RouterGroup) {
	var dbApi = v1.ApiGroupApp.CommonApiGroup.CommonDbApi
	router := Router.Group("commDb").Use(middleware.OperationRecord())
	routerNoRecord := Router.Group("commDb")
	{
		router.POST("quickEdit", dbApi.QuickEdit) // 更新 QuickEdit
		router.POST("updatePidSort", dbApi.UpdatePidSort)
	}
	{
		routerNoRecord.POST("pidData", dbApi.PidData)         // 获取 PidData
		routerNoRecord.POST("pidTreeData", dbApi.PidTreeData) // 获取 PidTreeData
		routerNoRecord.POST("getDict", dbApi.GetDict)
		routerNoRecord.POST("getDict2", dbApi.GetDict2)
		routerNoRecord.POST("getDictTree", dbApi.GetDictTree)
	}

}

// InitCommDbRouter 初始化 QuickEdit 路由信息
func (s *CommonDbRouter) InitCommonDbRouterPub(Router *gin.RouterGroup) {
	//var dbApi = v1.ApiGroupApp.CommonApiGroup.CommonDbApi
	//router := Router.Group("commDb").Use(middleware.OperationRecord())

	{

		//routerNoRecord.GET("test_db", dbApi.Test_db)
	}

}

File: web/src/api/common_db.ts
import { http,resData } from "@/utils/http";

 
export const pidData = (data?: object) => {
  return http.request<resData>("post", "/commDb/pidData", { data });
}; 


export const pidTreeData = (data?: object) => {
  return http.request<resData>("post", "/commDb/pidTreeData", { data });
};
  

export const getDictData = (data?: object) => {
  return http.request<resData>("post", "/commDb/getDict", { data });
};
 
 

export const getDictData2 = (data?: object) => {
  return http.request<resData>("post", "/commDb/getDict2", { data });
};
 
 


export const getDictTreeData = (data?: object) => {
  return http.request<resData>("post", "/commDb/getDictTree", { data });
};
  

export const updatePidSort = (data?: object) => {
  return http.request<resData>("post", "/commDb/updatePidSort", { data });
};
 